subtrack.MAUI/Shared/Components/BackButton.razor

```diff
@@ -0,0 +1,10 @@
+﻿@inject NavigationManager NavigationManager
+
+<button class="btn btn-outline-primary --bs-btn-padding-x: 2px; --bs-btn-padding-y: 2px;" @onclick="() => NavigationManager.NavigateTo(Url)">
```

Comment:
The "--" means its a variable so if you want to override that value you should do it in the style tag

Reply:
The rest are done, but I can't figure this one out while using the button css.

Reply:
What did you try?

Reply:
I don't recall. Even when I got changes I couldn't make sense of them. From only with changing to getting correct size but it's in the wrong place. I overrode numerous things from plain width to max width to the padding. I just don't understand the button css it seems.

Reply:
Ill try look into it when i have time

Reply:
Wont have time untill the end of the week, something you could try is copying the elements/styling to a html file with bootstrap imported so you can open the page in the browser and inspect the elements and check whats taking up all that space. Alternatively just ask chatgpt if you havent. Also try using !important if it doesnt work at first so you know what needs to be changed https://www.w3schools.com/css/css_important.asp

Reply:
I was looking into this and I tried something different. How do these buttons look like? 
![image](https://github.com/Code2Gether-Discord/subtrack/assets/7747808/8c9f76e3-26aa-4d49-9060-e01a8918334b)

and while pressing the button it looks like that
![image](https://github.com/Code2Gether-Discord/subtrack/assets/7747808/dd85fefe-d84f-4c81-83d9-dd2d4b9c65d2)

Let me know what you think!

Reply:
Looking good! I think itd be nice if the outline is always enabled so its consistent with other buttons and easily clickable. Except for that we just need to change the colors and we are good to go

Reply:
> Looking good! I think itd be nice if the outline is always enabled so its consistent with other buttons and easily clickable. Except for that we just need to change the colors and we are good to go

I guess what you meant is this?
![image](https://github.com/Code2Gether-Discord/subtrack/assets/7747808/e97adf3e-05cd-43ec-b7cc-f3067b9e3f13)

Reply:
Nice, maybe use success (green) on the + button

Reply:
Pushed the changes
![image](https://github.com/Code2Gether-Discord/subtrack/assets/7747808/c95503b9-808d-40ae-9637-0fc3f55809c3)